src/components/TooltipNew/style.scss

```diff
@@ -127,6 +139,18 @@
     visibility: visible;
   }
 
+  &--invert {
```

Comment:
You added styles for CssTooltip, but it's not used in the component. Check out `CssTooltip.js`

Reply:
ok